types/jsforce/jsforce-tests.ts

```diff
@@ -30,9 +41,9 @@ salesforceConnection.sobject("ContentVersion").create({
     }
 });
 
-salesforceConnection.sobject("ContentVersion").retrieve("world", {
+salesforceConnection.sobject<any>("ContentVersion").retrieve("world", {
     test: "test"
-}, (err: Error, ret: sf.Record) => {
+}, (err: Error, ret) => {
```

Comment:
now, the return type is able to be inferred and validated!

Reply:
nice :)